bundles/core/org.eclipse.smarthome.core.thing.xml/thing-description-1.0.0.xsd

```diff
@@ -46,12 +46,14 @@
 
     <xs:complexType name="channelType">
         <xs:sequence>
-            <xs:element name="item-type" type="xs:string"/>
+            <xs:element name="item-type" type="xs:string" minOccurs="0"/>
+            <xs:element name="trigger-type" type="xs:string" minOccurs="0"/>

```

Comment:
ideally we would introduce separate xs types for "normal" channels and "trigger" channels instead of having one with some elements or others depending on the actual channel type. however, this would break _all_ existing bindings, therefore I agree this is the best we can do without upsetting everybody.